src/main/java/frc/robot/RobotContainer.java

```diff
@@ -78,7 +78,6 @@ private void configureButtonBindings() {
     JoystickButton climbButton = new JoystickButton(m_driveJoystick, RobotMap.JOYSTICK_BUTTON.CLIMB);
 
     JoystickButton frontIntake = new JoystickButton(m_driveJoystick, RobotMap.JOYSTICK_BUTTON.FRONT_INTAKE);
```

Comment:
Please rename this button and RobotMap constant. "Front" is no longer needed.

Reply:
Resolved this in 9ca9bbb

Reply:
Resolved in 9ca9bbb and 4efc653